SolarSystemBreakfast_colorize.rb

```diff
@@ -0,0 +1,78 @@
+# Projest Solar System. Breakfast level
+require "colorize"
+
+class Planet
+  attr_accessor :name, :distance, :radius, :mass, :rotation
+
+  def initialize(planets)
+    @name = planets[:name]
+    @distance = planets[:distance]
+    @radius = planets[:radius]
+    @mass = planets[:mass]
+    @rotation = planets[:rotation]
+  end
+
+  def to_s
+   "Planet: #{@name} distance: #{@distance} radius: #{@radius} mass: #{@mass} rotation: #{@rotaion}"
+  end
+end
+
+
+class Solar_system
+attr_reader :planet
+
+  def initialize
+    @planet = {}
```

Comment:
Since `@planet` actually holds a bunch of planets, it could be clearer to call it `@planets`